components/automate-deployment/pkg/target/local_target.go

```diff
@@ -1344,9 +1344,26 @@ func (t *LocalTarget) installHabViaInstallScript(ctx context.Context, requiredVe
 		return err
 	}
 
+	// TODO(ssd) 2019-12-03: HACK until we remove the use of install.sh
+	lastBintrayVersion := "0.89.0"
```

Comment:
I think it should be something lower than `0.89.0`?

Reply:
@vsingh-msys I am not sure we can do anything about version 0.89. I think the script is just wrong for that version.

Reply:
It looks like `0.89.0` is actually a version that was released. `0.88` was followed by `0.90`

Reply:
quick tried with version `0.77.0` & `0.88.0` seems it is working file